hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/SubscriptionAssignmentRegistry.java

```diff
@@ -101,6 +94,38 @@ public SubscriptionAssignmentView createSnapshot() {
         return new SubscriptionAssignmentView(snapshot);
     }
 
+    private List<SubscriptionAssignment> readExistingAssignments() {
+        return subscriptionsCache.listActiveSubscriptionNames().stream().flatMap(subscriptionName -> {
```

Comment:
i think simple for-each would be more readable than this stream magic wit this long functions inside